dvc/stage.py

```diff
@@ -532,51 +505,52 @@ def create(
             logger.warning("Build cache is ignored when persisting outputs.")
             ignore_build_cache = True
 
-        if validate_state:
```

Comment:
I guess this one was a leftover from a very old initial pkg implementation and was not used anywhere else, right?

Reply:
yes, I noticed that no use of `create` assigns this value, so its always default

Reply:
Good catch @pared ! Thanks!